From 08acd75e627c6abc88e93ef1e7eda1ce6a306a84 Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Tue, 10 Jan 2012 18:24:46 +0530
Subject: [PATCH 492/609] mmc: hsmmc - remove invalid bus power setting

Original commit: 1a41d24c76d124113fb5759cbe9d2f29c1638baa

  From git://arago-project.org/git/projects/linux-am33x.git
  And ti-sdk-am335x-evm-05.05.00.00-Linux-x86-Install image

SD bus Power was setup at a wrong sequence inside irq code. This Commit
removes the same.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 drivers/mmc/host/omap_hsmmc.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/drivers/mmc/host/omap_hsmmc.c b/drivers/mmc/host/omap_hsmmc.c
index 81a809b..33da54f 100644
--- a/drivers/mmc/host/omap_hsmmc.c
+++ b/drivers/mmc/host/omap_hsmmc.c
@@ -1061,12 +1061,6 @@ static irqreturn_t omap_hsmmc_irq(int irq, void *dev_id)
 {
 	struct omap_hsmmc_host *host = dev_id;
 	int status;
-	struct omap_mmc_platform_data *pdata = host->pdata;
-
-	if (pdata->version == MMC_CTRL_VERSION_2) {
-		OMAP_HSMMC_WRITE(host->base, HCTL,
-			OMAP_HSMMC_READ(host->base, HCTL) | SDBP);
-	}
 
 	status = OMAP_HSMMC_READ(host->base, STAT);
 	do {
-- 
1.7.5.4

